dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -1550,6 +1612,91 @@ async fn cmd_nexus_blueprints_import(
     Ok(())
 }
 
+/// Runs `omdb nexus sagas list`
+async fn cmd_nexus_sagas_list(
+    client: &nexus_client::Client,
+) -> Result<(), anyhow::Error> {
+    // We don't want users to confuse this with a general way to list all sagas.
+    // Such a command would read database state and it would go under "omdb db".
+    eprintln!(
+        "{}",
+        textwrap::wrap(
+            "NOTE: This command only reads in-memory state from the targeted \
+            Nexus instance.  Sagas may be missing if they were run by a \
+            different Nexus instance or if they finished before this Nexus \
+            instance last started up.",
+            80
+        )
+        .join("\n")
+    );
+
+    let saga_stream = client.saga_list_stream(None, None);
+    let sagas =
+        saga_stream.try_collect::<Vec<_>>().await.context("listing sagas")?;
+
+    #[derive(Tabled)]
+    #[tabled(rename_all = "SCREAMING_SNAKE_CASE")]
+    struct SagaRow {
+        saga_id: Uuid,
+        state: &'static str,
```

Comment:
It would be nice if this included what saga it is, but it looks like the `internal_api::views::Saga` type doesn't currently include that information:

https://github.com/oxidecomputer/omicron/blob/c2ef4175e0638eab9302ad693927fa8942db03eb/nexus/types/src/internal_api/views.rs#L34-L43

We might want to add that? It doesn't have to be in this PR though.

Reply:
Also, if we add that, it could also be useful to list all sagas of a certain name, if I wanted to see, say, all the `instance-start` sagas. But, again, not directly related to the demo saga, and probably better suited for its own PR.

Reply:
Yeah.  I believe it's in the `dag` inside `steno::SagaView` but that's not currently exposed.  I think the reason is historical / it just wasn't stabilized.  This is doable but would just take some work because currently I think Steno throws away the parsed `dag`.  We'd probably want to save it in `steno::sec::Sec::do_saga_create()`.

Reply:
Yeah, we definitely don't need to do that in this branch, it would just be nice later.